authentication_service/helpers/account/auth_helpers.go

```diff
@@ -0,0 +1,32 @@
+package account
+
+import (
+	"errors"
+
+	"github.com/gin-gonic/gin"
+)
+
+func CheckUserType(c *gin.Context, role string) (err error) {
```

Comment:
this is currently unused. Also, this should'be have access to the whole `context` object. Rather, this should just have wrapper functions ( and have signature 

```go
func CheckUserType(actualUserType: string, desiredUserType: string) bool {
    // checks if the two are equal
}
```

the wrapper functions could then be 

```go
func isUserAdmin(actualUserType: string) bool {
    return checkUserType(actualUserType, "ADMIN")
}
```